app/views/thesis/select.html.erb

```diff
@@ -33,6 +33,25 @@
   </tbody>
 </table>
 
+<div id="publish-to-dspace" class="gridband" style="display: none;" aria-live="polite" role="region">
+  <div class="inline-action well">
+    <% if params[:graduation] == 'all' || params[:graduation].nil? %>
```

Comment:
I think, unlike almost all other similar parts of this app, we want the default to not be `all` here but an empty set. We should only publish one term at a time.

Reply:
We'll need to be careful to ensure the `publish_to_dspace` method checks for a degree param as well I think?